.github/workflows/core.yml

```diff
@@ -411,6 +411,11 @@ jobs:
         run: |
           rm -rf spark/interpreter/metastore_db
           ./mvnw verify -pl spark-submit,spark/interpreter -am -Dtest=org/apache/zeppelin/spark/* -Pspark-3.4 -Pspark-scala-2.13 -Phadoop3 -Pintegration -DfailIfNoTests=false ${MAVEN_ARGS}
+      - name: run spark-3.5 tests with scala-2.13 and python-${{ matrix.python }}
+        if: matrix.python >= '3.8'
+        run: |
+          rm -rf spark/interpreter/metastore_db
+          ./mvnw verify -pl spark-submit,spark/interpreter -am -Dtest=org/apache/zeppelin/spark/* -Pspark-3.5 -Pspark-scala-2.13 -Phadoop3 -Pintegration -DfailIfNoTests=false ${MAVEN_ARGS}
```

Comment:
BTW, hadoop2 support has been deprecated since Spark 3.4(no official binary artifacts, but allows user to compile by themselves), and removed since 3.5(totally removed the hadoop2 related stuff from codebase)

Reply:
I've been trying to remove hadoop2 for a while, but am still having problems with the Flink integration tests.
See #4674

Reply:
Remove Hadoop2 support would be a big change, I'm not very familiar with Flink, but for Spark, it requires at least Spark 3.x to support Hadoop 3, the Livy module still uses Spark 2.4.8.

And Spark moved to use Hadoop Shaded client since 3.2.0, which is a quite big change for Hadoop deps management.